www/js/diary/list.js

```diff
@@ -378,32 +378,6 @@ angular.module('emission.main.diary.list',['ui-leaflet',
       $state.go('root.main.detail');
     };
 
-    $scope.redirect = function(){
```

Comment:
Hm, longer-term, we should do a better structure for the current screen. Shouldn't have to touch diary to fix this.